task2/src/routes/user/index.js

```diff
@@ -0,0 +1,63 @@
+import express from 'express';
+
+import {
+    getUserById,
+    updateUser,
+    deleteUser,
+    createUser
+} from '../../usersData';
```

Comment:
You didn't learn this yet, but all data manipulation functions are usually gathered in services like UserService for example

Reply:
Would be done in task3